NavigationReactNative/src/ios/NVNavigationBarView.m

```diff
@@ -18,6 +18,16 @@ - (void)didSetProps:(NSArray<NSString *> *)changedProps
     [self.reactViewController.navigationController setNavigationBarHidden:self.hidden];
     if ([changedProps containsObject:@"title"]) {
         [self.reactViewController.navigationItem setTitle:self.title];
+    } else if ([changedProps containsObject:@"backTitle"]) {
+        NSInteger crumb = [self.reactViewController.navigationController.viewControllers indexOfObject:self.reactViewController];
+        UIViewController *previousController = crumb > 0 ? [self.reactViewController.navigationController.viewControllers objectAtIndex:crumb - 1] : nil;
+        if (previousController != nil) {
+            if (self.backTitle == nil) {
+                previousController.navigationItem.backBarButtonItem = nil;
+            } else {
+                previousController.navigationItem.backBarButtonItem = [[UIBarButtonItem alloc] initWithTitle:self.backTitle style:UIBarButtonItemStyleDone target:nil action:nil];
+            }
+        }
```

Comment:
We don't need to check if the `previousController` is nil. I think it reads better if we always reset the `backBarButtonItem` to nil and then only change it if the `backTitle` is not nil
```objective-c
previousController.navigationItem.backBarButtonItem = nil;
if (self.backTitle != nil) {
    previousController.navigationItem.backBarButtonItem = [[UIBarButtonItem alloc] initWithTitle:self.backTitle style:UIBarButtonItemStyleDone target:nil action:nil];
}
```
If you're happy with this change then please also apply it to the NVSceneController